app/components/date-time-picker.js

```diff
@@ -14,16 +16,38 @@ export default class DateTimePicker extends Component{
       this.minutes = this.date.getMinutes();
     }
   }
+
+  get datePickerLocalization() {
+    return {
+      buttonLabel: this.intl.t('auDatePicker.buttonLabel'),
```

Comment:
I didn't include the `placeholder` since that needs to stay in sync with the adapter, and we don't override that one. Ideally the adapter would support the correct formatting based on the locale as well but I don't know how to implement that yet and it's probably not worth the time investment (yet).